src/main/java/Duke.java

```diff
@@ -1,10 +1,244 @@
+                                    Deadline deadline = new Deadline(description, byTime);
+                                    lists.add(deadline);
+                                    Response msg = new Response(new Task[]{deadline}, Response.Tag.ADD, lists.size());
+                                    System.out.println(msg.getResponse());
+                                } catch (DateTimeParseException e) {
+                                    Response msg = new Response(new String[]{"Format of date and time is incorrect! Please fill in the date and time following the format below.",
+                                            "YYYY-MM-DDTHH:MM:SS"});
+                                    System.out.println(msg.getResponse());
+                                } catch (ArrayIndexOutOfBoundsException e) {
+                                    throw new MissingDeadlineException();
+                                }
+                            }
+                        } catch (ArrayIndexOutOfBoundsException e) {
+                            MissingDescriptionException realError = new MissingDescriptionException("a deadline");
+                            Response msg = new Response(new String[]{String.valueOf(realError)});
+                            System.out.println(msg.getResponse());
+                        } catch (MissingDeadlineException e) {
+                            Response msg = new Response(new String[]{String.valueOf(e)});
+                            System.out.println(msg.getResponse());
+                        }
+                    } else if (test[0].equals("delete")) {
+                        try {
+                            if (lists.size() == 0) {
+                                throw new IndexOutOfBoundsException();
+                            }
+                            if (test.length == 1) {
+                                throw new MissingNumberException();
+                            }
+                            int idx = Integer.parseInt(test[1]) - 1;
+                            Response msg = new Response(new Task[]{lists.get(idx)}, Response.Tag.REMOVE, lists.size() - 1);
+                            System.out.println(msg.getResponse());
+                            lists.remove(idx);
+                        } catch (IndexOutOfBoundsException e) {
+                            Response msg = new Response(new String[]{"The number does not exist!"});
+                            System.out.println(msg.getResponse());
+                        } catch (MissingNumberException e) {
+                            Response msg = new Response(new String[]{"Please select the task that you want to delete!"});
+                            System.out.println(msg.getResponse());
+                        }
+                    } else if (test[0].equals("find")) {
+                        try {
+                            if (test.length == 1) {
+                                throw new DukeException();
```

Comment:
Can consider printing a message to the user instead of using the default constructor.